### PR TITLE
feat: ssl auth compatibility

### LINKED ISSUE #6: compatibility with postgres ssl auth certificate
configure for compatibility with postgres ssl auth certificate

### PR DESCRIPTION
**Pull request by melancholiai**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

Closes #6

### CODE REVIEW

**Review by syncush — DISMISSED** [review]

File: helm/templates/configmap.yaml [hunk 1]
```diff
@@ -13,6 +13,15 @@ data:
   POSTGRES_HOST: {{ .dbConfig.host }}
   POSTGRES_DB: {{ .dbConfig.database }}
   POSTGRES_PORT: {{ .dbConfig.port | quote }}
+  {{- if .dbConfig.sslAuth.enabled }}
+  POSTGRES_ENABLE_SSL_AUTH: {{ "true" | quote }}
```
Comment (CptSchnitz) [inline helm/templates/configmap.yaml]: don't, just don't.

File: helm/templates/configmap.yaml [hunk 2]
```diff
@@ -13,6 +13,15 @@ data:
...
+  POSTGRES_CERTIFICATES_PATH: {{ .dbConfig.sslAuth.path }}
+  POSTGRES_SSL_CERT: {{ .dbConfig.sslAuth.path }}/{{ .dbConfig.sslAuth.certFileName }}
+  POSTGRES_SSL_KEY: {{ .dbConfig.sslAuth.path }}/{{ .dbConfig.sslAuth.keyFileName }}
+  POSTGRES_SSL_ROOT_CERT: {{ .dbConfig.sslAuth.path }}/{{ .dbConfig.sslAuth.caFileName }}
+  {{- else }}
+  POSTGRES_ENABLE_SSL_AUTH: {{ "false" | quote }}
```
Comment (CptSchnitz) [inline helm/templates/configmap.yaml]: same as before

**Review by CptSchnitz — CHANGES_REQUESTED** [review]

File: helm/templates/configmap.yaml (hunk 1 above)
Comment (melancholiai) [reply helm/templates/configmap.yaml]: done.

File: helm/templates/configmap.yaml (hunk 2 above)
Comment (melancholiai) [reply helm/templates/configmap.yaml]: done.

**Review by CptSchnitz — APPROVED** [review]

**Pull request merged**